AnimatedWordCloud/Animator/AnimationIntegrator.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2023 AnimatedWordCloud Project
+# https://github.com/konbraphat51/AnimatedWordCloud
+#
+# Licensed under the MIT License.
+"""
+Integrates the images into a single video (gif)
+"""
+
+from typing import List
+from PIL import Image
+
+
+def integrate_images(
+    image_paths: List[str],
+    filename: str,
+    duration: int = 500,
+) -> None:
```

Comment:
Add a simple test code like:
```python
assert integrate_images() == None
```

to check error

Reply:
OK